### PR TITLE
added FIDO2 security keys

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#6189 
Sources:
https://docs.microsoft.com/en-us/azure/active-directory/authentication/concept-authentication-passwordless
https://renenyffenegger.ch/notes/Windows/registry/tree/HKEY_LOCAL_MACHINE/Software/Microsoft/Windows/CurrentVersion/Authentication/Credential-Providers/index

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Supported credential providers include:
 |Fingerprint | \{BEC09223-B018-416D-A0AC-523971B639F5}|
 |Facial Recognition | \{8AF662BF-65A0-4D0A-A540-A338A999D36F}|
 |Trusted Signal<br>(Phone proximity, Network location) | \{27FBDB57-B613-4AF2-9D7E-4FA7A66C21AD}|
+|FIDO2 security keys | \{F8A1793B-7873-4046-B2A7-1F318747F427}|
 
 >[!NOTE]
 >Multifactor unlock does not support third-party credential providers or credential providers not listed in the above table.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md]: add MarkDown indent marker compatibility spacing, please.
```suggestion
> [!NOTE]
> Multifactor unlock does not support third-party credential providers or credential providers not listed in the above table.
```

File: windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md [hunk 2]
```diff
@@ -72,6 +73,7 @@ The default credential providers for the **First unlock factor credential provid
 The default credential providers for the **Second unlock factor credential provider** include:
 * Trusted Signal
 * PIN
+* FIDO2 security key
 
 Configure a comma separated list of credential provider GUIDs you want to use as first and second unlock factors. While a credential provider can appear in both lists, remember that a credential supported by that provider can only satisfy one of the unlock factors. Listed credential providers do not need to be in any specific order.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md]:
```suggestion
Configure a comma-separated list of credential provider GUIDs you want to use as first and second unlock factors. While a credential provider can appear in both lists, remember that a credential supported by that provider can only satisfy one of the unlock factors. Listed credential providers do not need to be in any specific order. 
```

**Comment by mypil** [comment]
@mapalko - PR has been copy edited and is ready for final review. This is based on issue #6189.

Thank you.

cc: @kenwith

**Comment by illfated** [comment]
Thank you for the interesting points. I presume you will be commenting later if anything here can be reused, rephrased or otherwise changed to provide some form of support as you noted.

**Comment by mapalko** [comment]
I commented on the original issue to get more clarity from the poster. I will provide an update based on their response on what should be added.

**Pull request closed without merging**